apps/cookbook/src/app/showcase/modal-showcase/modal-showcase.component.ts

```diff
@@ -94,6 +94,14 @@ export class ModalShowcaseComponent {
       defaultValue: '',
       type: ['string | string[]'],
     },
+    {
+      name: 'canDismiss',
+      description: `(Optional) Determines whether or not a modal can be dismissed.
+ 
+      The canDismiss option takes a callback function that returns either a boolean or an AlertConfig. If an AlertConfig is returned an alert will appear when the user tries to dismiss the modal. If false is returned the modal cannot be dismissed by user-interaction or with ModalController.hideTopMost(). `,
+      defaultValue: 'undefined',
```

Comment:
Implicitly, this is `true` because it defaults to true on Ionics end.  
Consider letting that show here to avoid confusion about the default behavior.
```suggestion
      defaultValue: 'true',
```